rules/falco_rules.yaml

```diff
@@ -3150,7 +3222,13 @@
 # # Change rule to different port, then different process name, and test again that it fires.
 
 - rule: Outbound or Inbound Traffic not to Authorized Server Process and Port
-  desc: Detect traffic that is not to authorized server process and port.
+  desc: > 
```

Comment:
We have about 3 sandbox detections that based on how they created the logic expression they won't evaluate to true unless populating the lists. Not a big fan of it, we may want to change them to placeholder always_true macros? @darryk10 just so we actually comply with our own new guidelines?

Reply:
@darryk10 and I synced and we came to the conclusion that 4 rules (including this one) actually to do not conform to the criteria for sandbox according to the maturity framework guidelines.

We decide to assign the `maturity_deprecated` label to those 4 rules.